File: data/EXP01/go/2135/2135.output_3.go
func wordCount(startWords []string, targetWords []string) int {
    startFreq := make(map[string]int)
    for _, word := range startWords {
        startFreq[word]++
    }

    count := 0
    for _, target := range targetWords {
        for i := 'a'; i <= 'z'; i++ {
            newWord := target + string(i)
            if startFreq[newWord] > 0 {
                count++
                break
            }
        }
    }

    return count
}

File: data/EXP01/python/2135/2135.output_3.py
# @lc app=leetcode id=2135 lang=python3
from typing import List

class Solution:
    def wordCount(self, startWords: List[str], targetWords: List[str]) -> int:
        freqArray = [[0] * 26 for _ in range(len(startWords[0]))]
        
        for word in startWords:
            for i, char in enumerate(word):
                freqArray[i][ord(char) - ord('a')] += 1
        
        count = 0
        for word in targetWords:
            for i, char in enumerate(word):
                freqArray[i][ord(char) - ord('a')] -= 1
                if freqArray[i][ord(char) - ord('a')] < 0:
                    break
            else:
                count += 1
        
        return count